src/wp-admin/plugins.php

```diff
@@ -664,9 +664,9 @@
 		);
 		$errmsg .= ' ' . __( 'If you notice &#8220;headers already sent&#8221; messages, problems with syndication feeds or other issues, try deactivating or removing this plugin.' );
 	} elseif ( 'resuming' === $_GET['error'] ) {
-		$errmsg = __( 'Plugin could not be resumed because it triggered a <strong>fatal error</strong>.' );
+		$errmsg = sprintf( __( '%s Plugin could not be resumed because it triggered a <strong>fatal error</strong>.' ), $plugin );
```

Comment:
1. With the name now added, we won't need the word "Plugin" anymore.
2. This should have a translators comment above it like:
```php
/* translators: %s: The plugin's name, or file path if a name is not available. */
```
3. `$plugin` refers to the plugin's file rather than its name.
Try adding this on line 652:
```php
$plugin_data = get_plugin_data( trailingslashit( WP_PLUGIN_DIR ) . untrailingslashit( $plugin ) );
```
Then, instead of using `$plugin` in the `sprintf()` call, you can use `$plugin_data['Name'] ?? "<code>{$plugin}</code>"` which includes a fallback to the plugin's file path in case the name is not available, such as when the plugin no longer exists.

:1234: These points also apply to the change on line 669.

Reply:
@costdev Thanks for the review, I have implemented the suggestions.